.ci/scripts/lib/duplicate-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Script that parses maven output to find any tests that are run more than once.
+# Returns 1 when no tests are found (likely an error, for example if the maven output changes) or
+# if any test is run more than once. Otherwise return 0.
+
+# We are using files instead of variables so that we don't spam stdout too much when tracing is enabled.
```

Comment:
:wrench: I would propose to keep the same header as the `flaky-tests` script, e.g.

```bash
#######################################
# Parses Maven output to find if any tests was run more than once.
# Globals:
#   None
# Arguments:
#   1 - file containing the test outputs
#   2 - writable file which where the duplicate tests will be written
# Outputs:
#   Error message on STDERR if duplicate tests were found.
#   Error message on STDERR if no tests were found, indicating the function is broken
# Returns:
#   0 if no tests were run multiple times
#   1 if at least one test was run multiple times
#######################################
```